react-pr2/src/App.js

```diff
@@ -1,22 +1,55 @@
+import { useSelector } from 'react-redux';
 import { Redirect, Route, Switch } from 'react-router-dom';
 import Header from './components/Header/index';
+import NotFound from './components/NotFound';
 import Profile from './components/Profile';
 import CartFeature from './features/Cart';
 import ProductFeature from './features/Product';
 import Checkout from './features/Product/pages/Checkout';
 import DetailPage from './features/Product/pages/DetailPage';
+import Admin from './components/Admin';
 
 function App() {
+  const currentUser = useSelector((state) => state.user.current);
+  const isLoggedIn = currentUser.role;
+
   return (
     <div className="app">
-      <Header />
       <Switch>
-        <Redirect from="/home" to="/" exact />
-        <Route path="/" component={ProductFeature} exact />
-        <Route path="/products/:productId" component={DetailPage} />
-        <Route path="/cart" component={CartFeature} />
-        <Route path="/checkout" component={Checkout} />
-        <Route path="/account" component={Profile} />
+        {isLoggedIn !== 1 && (
```

Comment:
```suggestion
        {isLoggedIn !== ADMIN_ROLE && (
```
Nhưng ở đây phải check role chứ nhỉ. Còn isLoggedIn là true hoặc false mà